lost/src/main/java/com/mapzen/android/lost/internal/FusionEngine.java

```diff
@@ -112,6 +118,10 @@ public FusionEngine(Context context, Callback callback) {
     if (passiveInterval < Long.MAX_VALUE) {
       enablePassive(passiveInterval);
     }
+
+    if (prepLooper) {
+      Looper.loop();
```

Comment:
@sarahlensing we do still explicitly start/stop (and bind) the service at this time. However I'm not sure how using both start/stop and bind effects the lifecycle exactly.

https://github.com/mapzen/lost/blob/master/lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImpl.java#L40-L41

Seems like some more investigation around how the various methods of starting/stopping/binding the service behaves and also how play services behaves under the following conditions would be good:

1. App is backgrounded by user with system home button
2. App is exited by user with system back button
3. App is quit by user using recent apps menu swipe
4. App is force quit by user using "App info > Force Stop"
5. App is killed by system (is there a way to simulate/induce this behavior besides the above?)